### PR TITLE
Rocket 0.5

### PR DESCRIPTION
**Pull request by mateuszkj**
Supersedes #29

### CODE REVIEW

**Comment by svenstaro** [comment]
This is excellent. @GREsau could you take a look at this?

File: rocket-okapi/src/response/responder_impls.rs [hunk 1]
```diff
@@ -95,22 +108,24 @@ macro_rules! status_responder {
 status_responder!(Accepted, 202);
 status_responder!(Created, 201);
 status_responder!(BadRequest, 400);
-status_responder!(Unauthorized, 401);
-status_responder!(Forbidden, 403);
+// status_responder!(Unauthorized, 401);
+// status_responder!(Forbidden, 403);
```
Comment (williamdes) [inline rocket-okapi/src/response/responder_impls.rs:112]: why is this disabled, can you add a comment about it ?

**Review by williamdes — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -39,6 +37,13 @@ fn get_user(id: u64) -> Option<Json<User>> {
     }))
 }
 
+// You can tag your routes to group them together
+#[openapi(tag = "Users")]
+#[post("/user", data = "<user>")]
+fn create_user(user: Json<User>) -> Json<User> {
+    user
+}
+
```
Comment (EdgyEdgemond) [inline README.md:46]:
```
#[openapi]
#[rocket::post("/webhook/event", format = "json", data = "<event>")]
async fn event(event: Json<model::Event>) {
    println!("{:?}", event);
}
```
`Could not find Rocket route attribute. Ensure the #[openapi] attribute is placed *before* the Rocket route attribute. `

Installing from this commit, with an async method (0.5 rocket) I get the above error. Can't confirm I'm not at fault here :P

Comment (mateuszkj) [reply README.md:46]: You have to include `rocket::post`

```rust
use rocket::post;

#[openapi]
#[post("/webhook/event", format = "json", data = "<event>")]
async fn event(event: Json<model::Event>) {
    println!("{:?}", event);
}
```

Comment (EdgyEdgemond) [reply README.md:46]: That fixed it. Cheers.

**Comment by EdgyEdgemond** [comment]
Looking at last release (Mar 2020) and last merge (a year ago today). Is there a more actively supported version of this library?

**Comment by GREsau** [comment]
Yikes, it's been a year already? This is long past due a merge...

Many thanks to @mateuszkj, @ThouCheese, and everyone else involved in taking care of this! ❤️

**Comment by GREsau** [comment]
This is now published to crates.io as [0.7.0-alpha-1](https://crates.io/crates/rocket-okapi/0.7.0-alpha-1)

**Comment by williamdes** [comment]
> This is now published to crates.io as [0.7.0-alpha-1](https://crates.io/crates/rocket-okapi/0.7.0-alpha-1)

And [okapi 0.6.0-alpha-1](https://crates.io/crates/okapi/0.6.0-alpha-1)

Thank you for being reactive ! This is very important

I could migrate without any problems: https://github.com/mountains/permafrost/commit/4d53f997295d8839b10ae263ac59411a6bc1ea56

**Pull request merged**